### PR TITLE
update the links to GitHub Discussions

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update the links to GitHub Discussions (organization-wide)

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
If we are to do this change, could you update the other occurences of https://github.com/mdn/mdn-community/discussions as well? I don't have preferences for either, although the fact that https://github.com/mdn/mdn-community is used as the org-wide discussions repo does sound like a changeable implementation detail to me, so this is reasonable.

**Comment by yin1999** [comment]
> If we are to do this change, could you update the other occurences of https://github.com/mdn/mdn-community/discussions as well? 

I've replaced all the occurences (and modified the content to match [the current categories](https://github.com/orgs/mdn/discussions).

> I don't have preferences for either, although the fact that https://github.com/mdn/mdn-community is used as the org-wide discussions repo does sound like a changeable implementation detail to me, so this is reasonable.

As those links to mdn/mdn-community discussions have been redirected to organization-wide discussions. And the redirection would break the link to specific category. Maybe it's a better idea to update them (reducing redirection and could follow the link to specific categories).

File: files/en-us/mdn/community/discussions/index.md [hunk 1]
```diff
@@ -57,41 +57,30 @@ Check out the definition of each discussion category below so that you can start
...
+          href="https://github.com/orgs/mdn/discussions/categories/content"
           >Content</a
         >
       </td>
       <td>
-        Discussions related to the
-        <a href="https://github.com/mdn/content">content</a> on MDN Web Docs.
+        Discussions related to improvements to existing
+        <a href="https://github.com/mdn/content">content</a>,
+        ideas for new content, or any other content suggestions on MDN Web Docs.
```
Comment (Josh-Cena) [inline files/en-us/mdn/community/discussions/index.md:67]: New content requests should go to https://github.com/mdn/mdn. So you should update files/en-us/mdn/writing_guidelines/what_we_write/index.md as well.

Comment (yin1999) [reply files/en-us/mdn/community/discussions/index.md:67]: Do you mean we need to create an issue in [mdn/mdn](https://github.com/mdn/mdn/issues) to place a new content request (so I need to remove the description about "new content" here)?

Comment (Josh-Cena) [reply files/en-us/mdn/community/discussions/index.md:67]: Yes, and I guess that's why the "content request" category doesn't exist any more.

**Review by Josh-Cena — APPROVED** [review]
Great. Thanks!

**Pull request merged**